www/src/components/LayoutGenerator.tsx

```diff
@@ -140,7 +142,7 @@ ${columnsString.join('')}
         values for each column and see the output below.
       </p>
       <div className="form-inline mb-4">
-        <label htmlFor="num-cols-range mr-2">
+        <label className="mr-2" htmlFor="num-cols-range">
           Number of Columns {numColumns}
         </label>
         <Input
```

Comment:
[curious] Since the `Input` component is [deprecated](https://paragon-openedx.netlify.app/components/input/), should we migrate from `Input` to `Form.Control` or otherwise here? Does properly `Form.Control` support `type="range"` today (it's not currently depicted on [the docs site](https://paragon-openedx.netlify.app/components/form/form-control/))?

Reply:
Done